### ISSUE TITLE
Incorrect report of SA1001 (commas should be spaced correctly) in interpolated string

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#2469]
:memo: This is similar to #2289.

Input:

```csharp
line.Append($"{testResult.DisplayName, -75}");
```

Expected: 

SA1001 is not reported.

Actual:

SA1001 is reported, and a space is inserted after the comma.

**Comment by sharwell** [comment]
I mistakenly thought that the space after the comma was not allowed in this code. It turns out the current behavior is the expected behavior.

**Comment by THammond9** [comment]
The space is allowed but VS auto formatting will remove the space which will cause SA1001. I think this should be fixed as it has been stated that the auto format behavior is the desired behavior.

**Comment by sharwell** [comment]
@THammond9 The Visual Studio formatter is the preferred default for new rules (where we haven't set a policy), but I'm not sure it demands a special case for a previously existing rule. I'm also not sure it doesn't.

**Comment by THammond9** [comment]
I write code with the PPT format document on save feature enabled. Not fixing this would be seriously annoying for me as I would have to do a build to save a file with this format in it. More importantly, Microsoft seems to have made it clear that this is the desired format in interpolated strings. But we seem to be saying that Microsoft's format is completely wrong and instead it should be done another way (causing incompatibilities in the process.) For the user it is annoying and frustrating. They are caught between a rock (Microsoft) and a hard place (Stylecop) with no good solutions; disable a feature they love using or disable the Stytlecop rule. If there was a way to choose the format in VS then it wouldn't bother me as I could just change the setting. (This isn't meant to sound confrontational, just a passionate argument in favor of making this change.)

**Comment by sharwell** [comment]
@THammond9 Sure, just make sure to update the documentation of any affected rules to cover this case.

**Comment by sharwell** [comment]
This is currently blocked on dotnet/roslyn#24499.
